Dockerfile

```diff
@@ -47,14 +47,14 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
 RUN apt-get update && apt-get install -y --no-install-recommends  \
     build-essential       \
     bzip2                 \
+    erlang                \
```

Comment:
Why do we need this? I think conda for rabbitmq will take care of the correct version of erlang and apt install of rabbitmq-server for arm64 will install this with required version https://www.rabbitmq.com/install-debian.html#apt-quick-start-cloudsmith.

Reply:
Yes, that is what I was hoping for. But it didn't work, unfortunately :(. I will create an issue about that hack.

Reply:
here is the issue: https://github.com/aiidateam/aiida-prerequisites/issues/58